src/rust/engine/src/graph.rs

```diff
@@ -94,11 +94,11 @@ impl Entry {
     match res {
       Ok(nr) =>
         Ok(
-          nr.clone().try_into().unwrap_or_else(|_| {
+          (*nr).clone().try_into().unwrap_or_else(|_| {
             panic!("A Node implementation was ambiguous.")
           })
         ),
-      Err(failure) => Err(failure.clone())
+      Err(failure) => Err((*failure).clone())
```

Comment:
no longer necessary after latest commit in https://github.com/alexcrichton/futures-rs/pull/442?